eng/pipelines/templates/jobs/publish-coverage.yml

```diff
@@ -0,0 +1,72 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: Artifacts
+    type: object
+    default: []
+  - name: DependsOn
+    type: string
+    default: ''
+  - name: RunCoverage
```

Comment:
Is this interesting any more? This step isn't doing any running but only publishing.

Reply:
Nope, removing